airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -315,7 +325,7 @@ def poll_job_completion(self, job_name, dependent_containers={'cloudsql-proxy'})
                             logging.info('killing dependent live container %s' % cname)
                             # there is a race condition between reading the status and trying to kill the running
                             # container. ignore the return code to duck the issue.
-                            subprocess.call(['kubectl', 'exec', pod['metadata']['name'], '-c', cname, 'kill', '1'])
+                            subprocess.call(['kubectl', '--namespace', 'airflow-{}'.format(configuration.get('core', 'environment_suffix')), 'exec', pod['metadata']['name'], '-c', cname, 'kill', '1'])
```

Comment:
Seems like the namespace parameterization should be extracted into a function. You use it several times and it will make these lines more readable.

Reply:
Done.